lib/apiservers/engine/backends/volume.go

```diff
@@ -98,13 +98,21 @@ func (v *Volume) VolumeRm(name string) error {
 	//FIXME: check whether this is a name or a UUID. UUID expected for now.
 	_, err := client.Storage.RemoveVolume(storage.NewRemoveVolumeParams().WithName(name))
 	if err != nil {
-		if _, ok := err.(*storage.RemoveVolumeNotFound); ok {
+
+		switch err := err.(type) {
+
+		case *storage.RemoveVolumeNotFound:
 			return derr.NewRequestNotFoundError(fmt.Errorf("Get %s: no such volume", name))
+
+		case *storage.RemoveVolumeConflict:
+			return derr.NewRequestConflictError(fmt.Errorf("Volume '%s' is in use", name))
+
+		case *storage.RemoveVolumeInternalServerError:
```

Comment:
This is the one we get if the volume store isn't specified and there's no default? What's the actual message that's sent back in that case?